packages/runner/src/pages/Runner/index.tsx

```diff
@@ -21,7 +21,11 @@ function getOfficeJsUrlToLoad(): string {
 
 const Runner = () => (
   <AwaitPromiseThenRender
-    promise={addScriptTags([getOfficeJsUrlToLoad()]).then(() => Office.onReady())}
+    promise={addScriptTags([getOfficeJsUrlToLoad()]).then(() =>
+      ensureOfficeReadyAndRedirectIfNeeded({
+        isMainDomain: true /* true for the Editor */,
```

Comment:
1. is this really supposed to be true?
2. if most are true perhaps have it be isNotMainDomain and have it be optional

Reply:
Aaaaggh, you're totally right.  That's what I get for copy-pasting too liberally.  Thanks for the catch.  It should be false.